FUNCTION(test)
	MESSAGE(FATAL_ERROR "test was called!")
ENDFUNCTION()
